Source/WebCore/platform/graphics/cg/UTIRegistry.mm

```diff
@@ -132,9 +133,14 @@ String MIMETypeForImageType(const String& uti)
 
 String preferredExtensionForImageType(const String& uti)
 {
-ALLOW_DEPRECATED_DECLARATIONS_BEGIN
-    return adoptCF(UTTypeCopyPreferredTagWithClass(uti.createCFString().get(), kUTTagClassFilenameExtension)).get();
-ALLOW_DEPRECATED_DECLARATIONS_END
+    ALLOW_DEPRECATED_DECLARATIONS_BEGIN
+    String oldExtension = adoptCF(UTTypeCopyPreferredTagWithClass(uti.createCFString().get(), kUTTagClassFilenameExtension)).get();
+    ALLOW_DEPRECATED_DECLARATIONS_END
+
+    auto *type = [UTType typeWithIdentifier:uti];
+    String extension = type.preferredFilenameExtension;
+    RELEASE_ASSERT(oldExtension == extension);
```

Comment:
Do we really want to `RELEASE_ASSERT` here?

Reply:
I was doing this for local testing to convince myself that the two API's give the same result. Since all tests passed with this, I can remove it now.